### PR TITLE
Design Refresh

### PR DESCRIPTION
**Pull request by ademagic**
Includes css cleanup, CSS SMDS logo and general design refresh.

Had this going in 3 commits for ease of read, but they all came up under my work github account, so I had to reset + squash.
### Before:

screen shot 2015-02-03 at 9 36 09 pm
### After:

screen shot 2015-02-03 at 9 38 46 pm

Check it out @brendanmurty @theromulans

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
+				</div>
+				<ul class="links">
+					<li><a href="http://twitter.com/schnitmydadsays">Twitter</a></li>
+					<li><a href="http://facebook.com/schnitmydadsays">Facebook</a></li>
+					<li><a href="https://market.android.com/details?id=com.schnitzel">Android</a></li>
+				</ul>
+			</div>
+			<footer>
+				<p>Web development by <a href="http://brendanmurty.com/">Brendan Murty</a> and <a href="https://github.com/brendanmurty/schnitmydadsays">shared on GitHub</a>.</p>
+				<p>Delicious background image courtesy of <a href="http://www.bonappetit.com/" target="_blank">Bonappetit.com</a>
```
Comment (ademagic) [inline index.html]: I actually have zero "courtesy" from bonappetit. Leave it up till someone yells at us?

File: assets/css/main.css [hunk 2]
```diff
@@ -1,73 +1,182 @@
...
+    font-family: 'PT Serif', serif;
+    font-size: 2.5em;
+    line-height: 1;
+    margin: 0 auto;
+    padding: 1.6em 1em 1.3em;
+    text-align: center;
+    width: 5em;
+    border: 4px solid #fff6c2;
+    box-shadow: 0px 0px 1px #eb362d, -1px 0px 1px #eb362d, 1px 0px 1px #eb362d, 0px -1px 1px #eb362d, 0px 1px 1px #eb362d;
+}
```
Comment (ademagic) [inline assets/css/main.css]: Tried to get as close as I could to Peter's designed logo without resorting to svgs. Plus I'm sure he'll have ideas on a redesign.

**Comment by theromulans** [comment]
Approved.

**Comment by ademagic** [comment]
Also took the liberty of running CSSComb through the css, to make it more readable.

**Comment by theromulans** [comment]
Libby.

File: assets/css/main.css [hunk 3]
```diff
@@ -1,73 +1,182 @@
-/* Reset */
```
Comment (ademagic) [inline assets/css/main.css:1]: The resets were fat and unnecessary for the most part. I got rid of em.

File: assets/css/main.css (hunk 2 above)
Comment (theromulans) [inline assets/css/main.css]: Works as a step in the right direction mate, thanks.

File: index.html (hunk 1 above)
Comment (theromulans) [inline index.html]: Not fussed either way. We should try and get some better shots of Schnits ourselves potentially. Then again, most Schnits we have are likely at night and/or in low light conditions.

Comment (ademagic) [inline index.html]: sounds like a schnit-off + product photography sesh is in order maybe? Potentially also grabbing some off of venues with high-scoring reviews

**Comment by ademagic** [comment]
Tested in IE9+, chrome, firefox and chrome for ipad.
I'm probably gonna add a few more mods for smaller screens if this PR stays open any longer.
@brendanmurty I can't see where to add issues to this repo, is that something you have to enable?

**Comment by bcm-works** [comment]
@ademagic Thanks mate, solid improvements here. Let's find a royalty free image or use our own instead of your proposed image. Feel free to improve this further as you see fit, no rush!

**Comment by ademagic** [comment]
@brendanmurty can you get peefor to review it? Live version lives here: http://miko.ademagic.com/schnitmydadsays/ -- maybe he can suggest some more quick improvements?

**Comment by bcm-works** [comment]
@ademagic Just forwarded that last comment to Peter, I'm sure he'll have something to say about it!

**Comment by bcm-works** [comment]
Let's remove the footer as part of these changes.

When we find a suitable image that we've taken over the years, we won't need the blurb about that.

Also, I don't need the credit for developing, as it's a team effort now and likely distracts visitors from what matters: Schnitzel reviews.

Any thoughts @ademagic @theromulans?

_I've also enabled [issues](https://github.com/brendanmurty/schnitmydadsays/issues) for this repository so we can track any future bugs or ideas_

**Comment by theromulans** [comment]
Both those points sound logical to me mate.

**Comment by bc-miko-ademagic** [comment]
Thanks for adding issues @brendanmurty – I've updated the branch with the changes you've suggested. I've also added the smds svg logo that Peter provided. Changes are live at my repo's project page. Looks a bit empty right now.

**Comment by bc-miko-ademagic** [comment]
screen shot 2015-02-10 at 12 45 00 pm

**Comment by bcm-works** [comment]
Thanks @ademagic, now we just need a decent schnitzel image from our escapades!

**Comment by bcm-works** [comment]
Here are some of my favourite images from our Facebook page:

65063_609273839119334_190882334_n
319859_491424187570967_1362362843_n
971272_707429879303729_3308208107950526912_n
1003926_662811890432195_593701662_n

If the image is stretching to fill the screen size, they might not work though.

@ademagic @theromulans

**Comment by ademagic** [comment]
just realised I commented on this PR with my bigcommerce account. Doh.

**Comment by bcm-works** [comment]
@ademagic Could you apply one of the images from my previous comment to the PR?

Let's get this live!

**Comment by ademagic** [comment]
Fixes are in. Had Sladj help me pick the background image, we really need more nice photos of schnitzel. Or, in Sladj's words, "You guys need photos of more schnitzel and less salad".

Word.
:eyes: @brendanmurty @theromulans

**Comment by bcm-works** [comment]
Thanks Miko, these changes are now live! http://schnitmydadsays.com/

@ademagic @theromulans

**Pull request merged**